tcf_website/admin.py

```diff
@@ -51,7 +51,7 @@ class SectionAdmin(admin.ModelAdmin):
     autocomplete_fields = ['instructors']
 
     def get_queryset(self, request):
-        qs = super(SectionAdmin, self).get_queryset(request)
+        qs = super().get_queryset(request)
```

Comment:
Not sure what these args were for...is this just a linting change? you sure we don't need those?

Reply:
Yeah pylint throws an error if it isn't set up like this

Reply:
@j-alicia-long  Yeah, there were some "errors" that were caught by pylint before I started working on ESLint. Seems like someone merged unlinted changes...